tests/test_redirect.py

```diff
@@ -62,10 +66,31 @@ def test_redirect_community_to_contribute(self, mozwebqa):
         Assert.contains("/contribute/", response.url)
 
     @pytest.mark.nondestructive
-    def test_redirect_firefox_mobile_to_mobile(self, mozwebqa):
+    def test_redirect_mobile_notes_to_android_notes(self, mozwebqa):
+        self._test_get_redirect(mozwebqa,
+                                '/mobile/notes/',
+                                '/firefox/android/notes/',
+                                False)
+        self._test_get_redirect(mozwebqa,
+                                '/mobile/beta/notes/',
+                                '/firefox/android/beta/notes/',
+                                False)
+        self._test_get_redirect(mozwebqa,
+                                '/mobile/aurora/notes/',
+                                '/firefox/android/aurora/notes/',
+                                False)
+        self._test_get_redirect(mozwebqa,
+                                '/mobile/37.0/releasenotes/',
+                                '/firefox/android/37.0/notes/',
```

Comment:
Ugh, `global.conf` is right but this test is just wrong. It should be `/firefox/android/37.0/releasenotes/`, of course. Will fix it shortly.